### PR TITLE
Use full-width image in single view

### PR DESCRIPTION
**Pull request by austinpray**
closes https://github.com/austinpray/kaori/issues/112

## before

image

## after

image

### CODE REVIEW

File: kaori/plugins/gacha/tui.py [hunk 1]
```diff
@@ -14,9 +14,75 @@
...
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": '\n'.join([
+                    f"*{'(no name)' if card.name.startswith(tmp_prefix) else card.name}*",
+                    f"_{card.engine.subtitle}_" if card.engine else '(stats pending)',
+                    card.description if card.description else '_(no description)_'
+                ])
+            },
+            # "accessory": {
```
Comment (ridhoq) [inline kaori/plugins/gacha/tui.py:34]: need this?

Comment (austinpray) [reply kaori/plugins/gacha/tui.py]: rebased

File: kaori/plugins/gacha/tui.py [hunk 2]
```diff
@@ -268,7 +296,7 @@ def battle_blocks(attacker: Card, defender: Card, battle_url: str):
                 "type": "button",
                 "text": {
                     "type": "plain_text",
-                    "text": "Watch Battle",
+                    "text": "Start Battle",
```
Comment (ridhoq) [inline kaori/plugins/gacha/tui.py:299]: I think you changed this in other PR

Comment (austinpray) [reply kaori/plugins/gacha/tui.py:299]: ffs I cannot do merges on my laptop screen

```suggestion
                    "text": "Watch Battle",
```

**Pull request merged**